Source/BugsnagEvent.m

```diff
@@ -569,6 +569,11 @@ - (instancetype)initWithUserData:(NSDictionary *)event {
     obj.context = bugsnagPayload[@"context"];
     obj.groupingHash = bugsnagPayload[@"groupingHash"];
     obj.error = [obj getMetadataFromSection:BSGKeyError];
+
+    if ([errors count] > 0) {
+        BugsnagError *bugsnagError = errors[0];
+        obj.customException = BSGParseCustomException(event, bugsnagError.errorClass, bugsnagError.errorMessage);
```

Comment:
Should the two strings be copied like in `initWithKSCrashData`? Also, can you rename `bugsnagError` to something else as this implies to me that it's some sort of internally generated construct.

Reply:
I don't see any reason why `initWithKSCrashData` needs to copy the string - I assume this is just done for legacy reasons. Renamed variable.